css_stylesheet.css

```diff
@@ -65,54 +65,60 @@ header {
 
 h2 {
 	text-align: right;
-	padding-top: 20px;
+	padding-top: 1em;
 	padding-right: 25px;
 	color: #303030;
+	font-size: 1.4em;
 }
 
 .aboutMeContent {
-	margin: 10px;
+	margin: 0.4em;
 	flex-direction: row;
 }
 
 .aboutMeText {
 	font-family: "Arial", sans-serif;
 	color: #343A40;
 	overflow: auto;
-	border-bottom: solid 3px #343A40;
-	border-top: solid 3px #343A40;
-	padding: 20px 5px;
+	border-top: solid 0.1em #343A40;
+	border-bottom: solid 0.1em #343A40;
+	padding: 0.5em 0.25em;
 	text-align: justify;
-	min-width: 200px;
-	/*font needs changing*/
 }
 
-.aboutMeText:hover {
+.aboutMeText:hover,
+.aboutMeText:active {
  	color: #000000;
 }
 	
 h3 {
 	text-align: right;
-	font-size: 25px;
-	padding-left: 25px;
-	padding-top: 30px;
+	font-size: 1.6em;
+	padding-left: 0.9em;
+	padding-top: 0.8em;
 	color: #303030;
 }	
 	
 .projectButtonsContainer {
-	display: flex;
+	min-width: 100%;
 }
 
 .portfolioButtons {
-	width: 24.3%;
-	flex-direction: row;
-	margin-top: 10px;
-	padding: 100px;
+	box-sizing:border-box;
+	display: inline-block;
+	margin-top: 0.4em;
+	padding: 7rem 0rem;
+	width: 25%;
 	background-color: #5C7CFA;
-	border: solid 1px #343A40;
-	font-size: 25px;
+	border: solid 0.01em #343A40;
```

Comment:
I know I said to use em, but borders are an occasion where px is fine. Actually, using ems might mean that the border disappears or becomes almost invisible if the base font size gets very small.